arelle/ValidateXbrl.py

```diff
@@ -53,14 +53,14 @@ class ValidateXbrl:
 
     authority: str
     authParam: dict[str, Any]
-    domainMembers: list
+    domainMembers: set = set()  # concepts which are dimension domain members
     extensionImportedUrls: set
     hasExtensionCal: bool
     hasExtensionDef: bool
     hasExtensionLbl: bool
     hasExtensionPre: bool
     hasExtensionSchema: bool
-    primaryItems: list
+    primaryItems: set = set()  # concepts which are line items (should not also be dimension members
```

Comment:
Why do these need to be initialized?

Reply:
Didn't realise it could be done from `Dimensions.py`. I'll revert.